apisix/plugins/kafka-proxy.lua

```diff
@@ -0,0 +1,63 @@
+--
+-- Licensed to the Apache Software Foundation (ASF) under one or more
+-- contributor license agreements.  See the NOTICE file distributed with
+-- this work for additional information regarding copyright ownership.
+-- The ASF licenses this file to You under the Apache License, Version 2.0
+-- (the "License"); you may not use this file except in compliance with
+-- the License.  You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+local core = require("apisix.core")
+
+
+local schema = {
+    type = "object",
+    properties = {
+        sasl = {
+            type = "object",
+            properties = {
+                username = {
+                    type = "string",
+                    default = "",
```

Comment:
has `minLength`, `maxLength` limit?

Reply:
reference these issue and PR, kafka supports unlimited length of username and password

https://github.com/edenhill/librdkafka/issues/1691
https://github.com/edenhill/librdkafka/commit/ff629e18e2431f1f52fedf5e411278c674774442

And our dependency `lua-resty-kafka` also does not enforce a limit on its length.